### ISSUE TITLE
Interactive Table Editor does not embedd

### ISSUE THREAD

**Issue by realB12** [RickStrahl/MarkdownMonster#884]
I can activate the interactive table editor, can edit great tables inside the table editor - all fine - but when clicking the top green button to finally embed my work into an active *.md document nothing happens: the table editor remains open, nothing is copied to the file, no error message, nada. I can close the table editor and continue working normally. 

Have I missed something? 

I have never used the table editor before. So I do not know whether this is a new problem. 

I am on an up-to-date Windows 10 Pro machine with newest MM version.

**Comment by RickStrahl** [comment]
I can duplicate and definitely a bug. Not sure what happened as it's been working a little bit ago when a bunch of new work was done on the editor.

**Comment by RickStrahl** [comment]
Well this is embarrasing - last update during all the table editor enhancements I refactored out a bunch of the command objects that were used to handle various operations and moved them into a different class. The main command for embedding unfortunately got left behind and the missing command never any tests as automated closing works. 

I've updated in 2.1.7.7 that has the fix. Also in current version Ctrl-Enter in the entry fields should work hitting the same code path.
